modules/ossm-accessing-bookinfo-application-using-gateway-api.adoc

```diff
@@ -0,0 +1,51 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-accessing-bookinfo-application-using-gateway-api"]
+= Accessing the Bookinfo application by using Gateway API
+:context: ossm-accessing-bookinfo-application-using-gateway-API
+
+In {ocp-product-title} 4.15 and later versions, the Gateway API CRDs are not available by default
+and must be enabled in order to be used. 
+
+. Enable the Gateway API CRDs:
++
+[source,terminal]
+----
+$ oc get crd gateways.gateway.networking.k8s.io &> /dev/null ||  { oc kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v1.0.0" | oc apply -f -; }
+----
+
+. Create and configure a gateway using a `Gateway` resource and `HTTPRoute` resource:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/gateway-api/bookinfo-gateway.yaml
```

Comment:
Since the "deploying bookinfo" example deploys bookinfo into the `bookinfo` namespace, you'll need to deploy the gateway in the bookinfo namespace as well:
```suggestion
$ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/gateway-api/bookinfo-gateway.yaml -n bookinfo
```

Reply:
done